scipy/stats/tests/test_distributions.py

```diff
@@ -445,6 +445,22 @@ def test_mean(self):
         x = stats.chi.mean(df=1000)
         assert_allclose(x, self.CHI_MEAN_1000, rtol=1e-12)
 
+    # Entropy references values were computed with the following mpmath code
+    # import mpmath as mp
```

Comment:
If you are going to set `dps` with `mp.dps = 50` as in the next line, then you must *import* the `mp` namespace from `mpmath`.  That is, this line should be `from mpmath import mp`.  Otherwise, the line `mp.dps = 50` sets the value of the `dps` attribute in (renamed) `mpmath` namespace, which has no effect.  So this needs to be fixed, and then the results computed with `mpmath` and used in the unit tests need to be recomputed.

Inspired by this PR, I added the `entropy` function to `beta`, `chi`, `chi2` and `gamma` in [`mpsci`](https://github.com/WarrenWeckesser/mpsci).  Then, with `mpsci`, I get, for example,

```
In [1]: from mpmath import mp

In [2]: mp.dps = 50

In [3]: from mpsci.distributions import chi

In [4]: float(chi.entropy(500))
Out[4]: 1.0720309422146606

```

Reply:
Thanks for finding this one, have to fix that also in my Von Mises-Fisher PR.